SPECS/moby-runc/moby-runc.spec

```diff
@@ -1,22 +1,23 @@
 Summary:        CLI tool for spawning and running containers per OCI spec.
 Name:           moby-runc
-Version:        1.0.0~rc10+azure
-Release:        6%{?dist}
+Version:        1.0.0~rc94+azure
```

Comment:
Is this what the moby upstream team is supporting?

Reply:
Yes this is the latest